modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v8_8_2/SchemaUpgradeProcess.java

```diff
@@ -24,7 +24,11 @@ protected void doUpgrade() throws Exception {
 				SQLTransformer.transform(
 					StringBundler.concat(
 						"select ObjectField.dbColumnName, ",
-						"ObjectField.dbTableName from ObjectField where ",
+						"ObjectField.dbTableName from ObjectField inner join ",
+						"ObjectDefinition on ",
+						"ObjectDefinition.objectDefinitionId = ",
+						"ObjectField.objectDefinitionId where ",
+						"ObjectDefinition.status != 2 and ",
```

Comment:
Can we use `WorkflowConstants.STATUS_DRAFT` instead of `2`?

Reply:
Done. Thanks!